dev/com.ibm.ws.ui.tool.serverConfig/resources/WEB-CONTENT/js/components/core.js

```diff
@@ -9,909 +9,753 @@
+     };
+     
+     
+     var retrieveServerData = function(forceServerRefresh) {
+         var deferred = new $.Deferred();
+         if(!serverDataInitialized || forceServerRefresh) {
+             var retrieveServerWritePathsPromise = fileUtils.retrieveServerWritePaths();
+             var retrieveServerVariableValuesPromise = fileUtils.retrieveServerVariableValues();
+             var retrieveSchemaFilePromise = fileUtils.retrieveSchemaFile();
+             var retrieveFeatureListPromise = fileUtils.retrieveFeatureList();
+             var retrieveValidatorSchemaPromise = validationUtils.retrieveSchema();
+             $.when(retrieveServerWritePathsPromise, retrieveServerVariableValuesPromise, retrieveSchemaFilePromise, retrieveFeatureListPromise, retrieveValidatorSchemaPromise).done(function(serverWritePaths, serverVariableValues, schemaFile, serverFeatureList, validatorMetaDataObject) {
+                 serverDataInitialized = true;
+                 serverSchemaInTextFormat = schemaFile;
+                 featureList = serverFeatureList;
+
+                 deferred.resolve();
+             }).fail(function(jqXHR) {
+                 errorOnLastLoad = true;
+                 deferred.reject(jqXHR);
+             });
+         } else {
+             deferred.resolve();
+         }
+         return deferred;
+     };
+ 
+     
+     var updateNavigation = function(hash) {
+         if(embedded !== constants.EMBEDDED_EXPLORE_TOOL) {
+             window.top.history.pushState(null, null, hash);
+         }
+     };
+ 
+     
+     var getFileURL = function(filePath) {
+         var baseURL = window.top.location.href.split("#")[0];
+         if(embedded === constants.EMBEDDED_EXPLORE_TOOL) {
+             if(environment === constants.ENVIRONMENT_COLLECTIVE && serverId !== null && serverId !== undefined) {
+                 return baseURL + "#" + constants.EXPLORE_TOOL_HASH  + "/" + constants.EXPLORE_TOOL_SERVERS_SEGMENT + "/" + serverId + "/" + constants.EXPLORE_TOOL_CONFIGURE_SEGMENT + "/" + filePath;
+             } else {
+                 return baseURL + "#" + constants.EXPLORE_TOOL_HASH + "/" + constants.EXPLORE_TOOL_CONFIGURE_SEGMENT + "/" + filePath;
+             }
+         } else {
+             var hash = "#" + constants.SERVER_CONFIG_HASH + "/";
+             if(environment === constants.ENVIRONMENT_COLLECTIVE && serverId !== null && serverId !== undefined) {
+                 hash = hash + serverId + "/";
+             }
+             return baseURL + hash + filePath;
+         }
+     };
+ 
+     
+     var setFile = function(filePath) {
+         if(embedded === constants.EMBEDDED_EXPLORE_TOOL) {
+             if(environment === constants.ENVIRONMENT_COLLECTIVE) {
+ //                window.top.history.pushState(null, null, "#" + constants.EXPLORE_TOOL_HASH  + "/" + constants.EXPLORE_TOOL_SERVERS_SEGMENT + "/" + serverId + "/" + constants.EXPLORE_TOOL_CONFIGURE_SEGMENT  + "/" + filePath);
```

Comment:
Do we need to keep the commented out code?

Reply:
Resolving because this is not new code

Reply:
Commented line removed.